modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -85,7 +85,7 @@ spec:
 ----
 <1> The name of the log forwarding CR must be `instance`.
 <2> The namespace for the log forwarding CR must be `openshift-logging`.
-<3> Set to `enabled` to enable log forwarding.
+<3> Set to `true` to enable log forwarding to the specified outputs. 
```

Comment:
Looks like this was missed from the previous change `<2> Parameter to disable the default log forwarding behavior.`